metacat-connector-snowflake/src/main/java/com/netflix/metacat/connector/snowflake/SnowflakeConnectorTableService.java

```diff
@@ -113,4 +123,28 @@ public void rename(@Nonnull final ConnectorRequestContext context,
     public boolean exists(@Nonnull final ConnectorRequestContext context, @Nonnull final QualifiedName name) {
         return super.exists(context, name.cloneWithUpperCase());
     }
```

Comment:
We can make this as an explicit function to provide the name upperCaseConversion for snowflake, e.g. getSnowflakeTableNameString() instead of implicitly doing the changes using name.cloneWithUpperCase().

Reply:
Will do.